web/webpack.config.js

```diff
@@ -84,7 +87,21 @@ module.exports = (env) => {
             }),
             new webpack.DefinePlugin({'process.env.API_HOST': JSON.stringify(env.API_HOST)}),
             new webpack.DefinePlugin({'process.env.MOCK_API': JSON.stringify(env.MOCK_API)}),
+            new EncodingPlugin({encoding: 'iso-8859-1'})
         ],
-        devtool: "source-map"
+        devtool: "source-map",
+        optimization: {
```

Comment:
This is aids caused by all of the stuff that Katex injects into our built JS bundle